.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+    "extends": [
+	"eslint:recommended",
+	"plugin:react/recommended"
+    ],
+    env: {
+	browser: true,
+    },
+    parserOptions: {
+	ecmaVersion: 6,
+	ecmaFeatures: {
+	    jsx: true,
+	},
+	sourceType: 'module',
+    },
+    globals: {
+	gettext: false,
+	ngettext: false,
+	interpolate: false,
+	l: false,
+	expect: false,
+	test: false,
+	browser: false,
+	jest: false,
+	Promise: false,
+	require: false,
+	shortcut: false,
+	sorttable: false,
+        $: false,
+        Pontoon: false,
+        jQuery: false
```

Comment:
It's interesting to put this here. I would instinctively put them in each file instead, adding only the globals being used there. Maybe keep just a few exceptions here, like `Promise`. But I find there is value in having, at the top of each file, a list of the globals being used. It makes it easier to spot where something is coming from.

Reply:
im not sure i agree - esp for such things like new language primitives

Reply:
rereading, thats what you said re Promise